website/translated_docs/en-US/cita/advanced-use/contracts/rust.md

```diff
@@ -268,4 +268,4 @@ $ curl -X POST --data '{"jsonrpc":"2.0","method":"call", "params":[{"to":"0x0000
 {"jsonrpc":"2.0","id":2,"result":"0x0000000000000000000000000000000000000000000000000000000000000011"}
 ```
 
-符合我们编写的智能合约预期。
\ No newline at end of file
+符合我们编写的智能合约预期。
```

Comment:
should we keep trailing newline? @leeyr338

Reply:
> @angiie should not use Chinese as commit message

Got it!